### DISCUSSION TITLE
Linux HDR is coming, should we expect 10 bit/HDR support in the future?

### DISCUSSION THREAD

**Discussion by unsafeunaudited** [nroduit/Weasis#512] [General]
First off, I would like to thank you for developing Weasis. *I am really liking it*.

There seems to be an increase in the general availability of HDR600 capable monitors and OLED displays from the likes of LG, Dell and Asus (with the latter having Pantone validation).

But on the flip side, a lot of the freeware DICOM software available in Windows (notably Radiant), still does not support 10bit/HDR displays and last I've heard from their forum, the dev doesn't want to implement it because of various requirements.

With the imminent arrival of HDR support in Linux, I am a bit curious if there is a long term plan or at least an intent to support 10 bit displays, at the very least in the form of consumer HDR monitors? Honestly I really don't expect a yes *or even a maybe*, I just want to get this out of my head. Also because I am excited for HDR in Linux.

Thank you again.

**Comment by nroduit** [comment]
Having not investigated this subject, I'd like to understand the objective benefit of having a 10-bit screen with DICOM data and software like Weasis.

**Comment by unsafeunaudited** [comment]
Briefly, having a 10-bit chain - 10 bit capable GPU, OS, cable connectors, monitor and DICOM view application allows for:
- Better diagnostic accuracy as it allows to show the subtle 10-bits of gray levels (great for mammography studies).
- Better compatibility with the currently available medical grade displays.
- Conformance of the [section 14 of the DICOM Grayscale Standard Display Function](https://dicom.nema.org/medical/dicom/current/output/pdf/part14.pdf)

The counterpoint is that:
- I believe you have a disclaimer that Weasis is not considered a medical device, legally speaking. Conformance to standards is just nice to have (at this point at least).
- Most laptops are calibrated once only and medical grade displays are calibrated regularly with a built in sensor/calibrator along the frame of the monitor (as is the case for my laptop and desktop monitor - I still dont have a proper medical grade display).
- The ability to show 10-bits of information is just a part of what makes a monitor _medical grade_  other features are also required (and off my head I cannot recall the rest of the features).
- Windowing and Leveling can sort of mitigate the lack of 10-bit display capability.

Technically, you could apply it now even if Linux does not have HDR support yet because Windows 10+ already has native support for 10-bit displays.

Also as a final note I am not entirely sure about the nomenclature, so there may be a slight misunderstanding on the technical wording of it. It's been a while since I've brushed on imaging physics but if I recall correctly, 10-bit is the proper term that radiologists desire. I'm not certain if HDR is the same as 10 bit support or if one is a subset of another when it comes to implementation.

**Comment by nroduit** [comment]
Thanks for these explanations. However, I'm still not convinced that the majority of users can see a real difference. See this [article](https://www.popsci.com/humans-can-only-distinguish-between-about-30-shades-gray).

The windowing tools and the types of transfer functions (linear, sigmoid, logarithmic) already make it possible to navigate through the contrasts of 12- or 16-bit images to highlight them.

HDR technology is interesting for global rendering, but cannot replace windowing for highlighting structures during diagnosis.
